### ISSUE TITLE
Cloud build is passing for debug but release failing

### ISSUE THREAD

**Issue by natanraj** [eclipsesource/tabris-js#997]
I am seeing the Cloud build is passing with no code errors, but  the status is shown failed when signing the release packaging with below error

```
BUILD SUCCESSFUL
Total time: 1 mins 54.301 secs
Built the following apk(s): 
	/home/travis/build/eclipsesource/tabris-js-build-template/build/platforms/android/build/outputs/apk/android-release-unsigned.apk
```

`jarsigner: unable to sign jar: javax.net.ssl.SSLHandshakeException: Remote host closed connection during handshake`

**Comment by ralfstx** [comment]
Investigating ...

**Comment by ralfstx** [comment]
This was an issue with Java 7 using TLS 1.0 by default. Fixed by enforcing TLS 1.2 / 1.2.
